ui/src/app/login/page.tsx

```diff
@@ -1,23 +1,34 @@
 "use client";
 
-import { Button, TextField, Typography } from "@mui/material";
+import { Button, Divider, TextField, Typography } from "@mui/material";
 import { FormBox, ImageBox, LoginContainer, MainBox } from "@/components/login/LoginComponents";
 import CustomLink from "@/components/login/CustomLink";
+import PasswordField from "@/components/common/PasswordField";
 
 export default function LoginPage() {
     return (
         <MainBox>
             <LoginContainer>
                 <ImageBox/>
                 <FormBox>
-                    <Typography variant="h5" fontWeight="bold">
+                    <Typography variant="h4" fontWeight="bold">
                         Login To Your Account
                     </Typography>
                     <TextField label="Email" variant="outlined" size="small" fullWidth/>
-                    <TextField label="Password" variant="outlined" size="small" type="password" fullWidth/>
+                    <PasswordField label="Password" variant="outlined" size="small" fullWidth/>
                     <Button variant="contained" color="primary" fullWidth>
                         LOGIN
                     </Button>
+                    <Divider sx={{ width: "100%", my: 2 }}>Or log in with</Divider>
```

Comment:
For now comment this out, it's hard to integrate with google login without having an application.